web-components/src/components/form/BasketTakeForm.tsx

```diff
@@ -137,7 +137,11 @@ const BasketTakeForm: React.FC<React.PropsWithChildren<FormProps>> = ({
       onSubmit={submitHandler}
     >
       {({ values, submitForm, isSubmitting, isValid, submitCount, status }) => (
-        <Form>
+        <Form
+          placeholder={undefined}
```

Comment:
unrelated to this work but had to add that to remove typescript errors that were now firing due to some dep upgrades

Reply:
Why does the dep now require undefined everywhere? Seems like a bug

Reply:
yeah it was a strange dependency bug
while rebasing on `dev` and reinstalling dependencies, I could get rid of it